lib/js/atoms/Tile/Tile.vue

```diff
@@ -0,0 +1,156 @@
+<template>
+	<ripple-wrapper :disable="!interactive">
+		<div :class="{ '-interactive': interactive }" class="a-tile">
+			<icon
+				v-if="leftIcon"
+				:icon="leftIcon"
+				:size="ICON_SIZES.SMALL"
+				class="a-tile__leftIcon"
+			/>
+			<div class="a-tile__center">
+				<span class="a-tile__eyebrowText" v-text="eyebrowText" />
+				<span class="a-tile__text" v-text="text" />
+			</div>
+			<icon
+				v-if="interactive"
+				:icon="ICONS.FA_CHEVRON_RIGHT"
+				:size="ICON_SIZES.SMALL"
+				class="a-tile__interactiveIcon"
+			/>
+			<icon
+				v-else-if="rightIcon"
+				:icon="rightIcon"
+				:size="ICON_SIZES.SMALL"
+				class="a-tile__rightIcon"
+			/>
+			<div v-else-if="additionalText" class="a-tile__additionalText">{{
+				additionalText
+			}}</div>
+		</div>
+	</ripple-wrapper>
+</template>
+
+<style lang="scss" scoped>
+@import '../../../styles/settings/typography';
+@import '../../../styles/settings/spacings';
+@import '../../../styles/settings/colors/tokens';
+
+::v-deep .ripple {
+	background-color: $color-neutral-ripple !important;
+}
+
+.a-tile {
+	width: 100%;
+	min-height: $space-xl;
+	background-color: $color-neutral-background;
+	display: flex;
+	flex-direction: row;
+	align-items: center;
+	padding: $space-xxs $space-xs;
+
+	&.-interactive {
+		cursor: pointer;
+
+		&:hover {
+			background-color: $color-neutral-background-hovered;
+		}
+	}
+
+	&__leftIcon {
+		margin-right: $space-xs;
+	}
+
+	&__center {
+		display: flex;
+		flex-direction: column;
+		flex-grow: 1;
+	}
+
+	&__interactiveIcon {
```

Comment:
IMO we should use `__rightIcon` class with modifier `-interactive` to change the color. By doing that you group common styles in the same place, so there is no way they get out of synch